### ISSUE TITLE
Property 'issues' does not exist on type 'GitHub'

### ISSUE THREAD

**Issue by nwtgck** [actions/toolkit#335]
**Describe the bug**
Hi,

I had the following build error.
```
src/main.ts:40:28 - error TS2339: Property 'issues' does not exist on type 'GitHub'.

40         await githubClient.issues.createComment({
                              ~~~~~~


Found 1 error.
```

<img width="725" alt="image" src="https://user-images.githubusercontent.com/10933561/73824851-3dc95b80-483e-11ea-8e2e-4c2e039ea15f.png">

I use `@actions/github@2.1.0`. This bug may be a bug of  octoki/rest.js.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by hardboiled** [comment]
I was having similar issues on this. I found that my type definition file generated as part of the github actions package here:
`node_modules/@actions/github/lib/github.d.ts`

imports Octokit as a default instead of as a decomposed import:
```typescript
import { graphql as GraphQL } from '@octokit/graphql/dist-types/types';
import Octokit from '@octokit/rest';
import * as Context from './context';
export declare const context: Context.Context;
export declare class GitHub extends Octokit {
```
should be:
```
import { Octokit } from '@octokit/rest';
```

this breaks compilation for me. When I hand edit the package with this change, my project compiles.

This [octokit release](https://github.com/octokit/rest.js/releases/tag/v16.43.1) seems to be related to this issue.

I ended up nuking all my typescript and just using raw node instead, since I couldn't compile using the types supplied by the `@actions/github` package.

**Comment by nwtgck** [comment]
@hardboiled Thanks! That may relate to my PR: #332.

**Comment by thboop** [comment]
@nwtgck this should be resolved in `2.1.1`,  I've been unable to reproduce in that version.

I'm going to close this out for now, but if you are still encountering issues please let me know and we can revisit this item.

Thanks for your contribution!

**Comment by Demeanor60** [comment]
I ran into a similar issue and discovered an answer https://react-frame-component-pr.github.io/importer/
